html.c

```diff
@@ -742,11 +742,17 @@ void print_html_node(GString *out, node *n, scratch_pad *scratch) {
 		case DEFLIST:
 			pad(out,2, scratch);
 			scratch->padded = 1;
-			g_string_append_printf(out, "<dl>\n");
+            if ((n->children && n->children->key == ADMONITION)) {
```

Comment:
Looks like I need to fix some spacing here to maintain consistency.  I will do that in an update to this PR